brainframe/cli/print_utils.py

```diff
@@ -0,0 +1,102 @@
+import sys
+from enum import Enum
+from pathlib import Path
+import os
+
+import i18n
+
+
+class Color(Enum):
```

Comment:
Not a good idea to hardcode color values. There's probably a way to pull from the shell theme.

Reply:
Is that really necessary to invest time into the theming of colors in a CLI?

Reply:
These aren't RGB color values, they're ANSI escape codes that represent colors specified by the shell theme, so it's actually a pretty sweet idea to hardcode them: https://en.wikipedia.org/wiki/ANSI_escape_code#Colors

I'll add a comment saying what these are.

Reply:
Oh you're total right